manifests/t/TechPowerUp/GPU-Z/2.54.0/TechPowerUp.GPU-Z.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: TechPowerUp.GPU-Z
+PackageVersion: 2.54.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silentWithProgress
+InstallerSwitches:
+  SilentWithProgress: -installSilent
+  Interactive: -install
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://us5-dl.techpowerup.com/files/fxwYRaaiBk1zGISQ1KmL3A/1688446847/GPU-Z.2.54.0.exe
```

```suggestion
  InstallerUrl: https://us2-dl.techpowerup.com/files/GPU-Z.2.54.0.exe
```
I believe the current URL contains a hash that'll expire out quickly. This URL, although 404 in the browser, will allow package managers to use this as a static url